disco_aws_automation/version.py

```diff
@@ -1,5 +1,5 @@
 """Place of record for the package version"""
 
-__version__ = "2.4.12"
+__version__ = "2.4.13"
```

Comment:
if we go with semantic versioning (https://semver.org/), which is what we modeled our versioning scheme after many years ago (i'm pretty sure all of our teams follow this), then this should be bumped up to `2.5.0`

Reply:
i found it: http://wgencontractorwiki.aws.wgen.net/index.php/DPP/13.html
wow 2012... man i've been here a long time...

Reply:
Technically this would be `3.0.0` since the change is non-backwards compatible.